lib/main.dart

```diff
@@ -121,3 +94,38 @@ class Counter with ChangeNotifier {
     notifyListeners();
   }
 }
+
+class IconLogo extends StatelessWidget {
+  
+  @override 
+  Widget build(BuildContext context){
+    final screenWidth = MediaQuery.of(context).size.width;
+
+    return Card(
+      elevation: 5,
+      color: Colors.green,      // <== doesn't show up
+      shape: CircleBorder(
+        side: BorderSide(
+          color: Colors.deepPurple,    // TODO : to change to the backgroung color of the app
+          width: screenWidth / 50.0    // to tweak between this width and the border of the BoxDecoration
+        )
+      ),
+      child: Container(
+        // color: Colors.blue,        // <== CRASHES the app, the container can't have decoration and a color
+        width: screenWidth / 2.5,     // 1/2.5 of screen width
+        height: screenWidth / 2.5,    // <== required
```

Comment:
Height is only required as you are using the image as the background of the box decoration.
Once you remove the box decoration it won't be needed.